api/src/Controllers/AuthenticationController.php

```diff
@@ -178,13 +187,21 @@ private function processOneTimeUseTokens(): bool
                     $need_auth = false;
                     $this->dataLayer->deleteOneTimeUseToken($tokenId);
                 }
+                else
+                {
+                    $err = 'Authorisation token not valid for this URL.';
```

Comment:
I'm not sure that we would want to tell the user this. Perhaps we should, if a valid token is not found, return a more generic message to the user, e.g. "Invalid one-time authorization token, please go back and try again...", but log the reason internally.